README.md

```diff
@@ -42,21 +78,13 @@ You also need to download your user certificate to access the *Stratio GenAI Dev
 
    * `ca-cert.crt`: Certificate authority
    * `<user>.crt`: User certificate
-   * `<user>_private.key`: User key
-
-## Local development
+   * `<user>_private.key`: User private key
 
-### Clone the repository
+### Extra environment variables
 
-Clone the Git repository in your local machine:
+Some clients provided in *Stratio GenAI API*, like the *Stratio GenAI Gateway* client or the *Stratio GenAI API* client, obtain their certificates by reading some environment variables instead of accessing Vault. Some of them also expect some extra variables defined, which, in the "normal" execution mode inside *Stratio GenAI API*, are set by *Stratio GenAI API* itself. If your chain uses these clients, you should set the corresponding environment variables to run in locally.  
 
-```bash
-git clone https://github.com/Stratio/genai-examples.git
-```
-
-### Configure the environment variables
-
-For local development, you need to configure the user certificates to access the *Stratio GenAI Developer Proxy* service. You need the URL of the service and the path of the folder where you uncompressed the user certificates.
+For your convenience, we provide a script that generates an `.env` file with all the needed variables by just giving it the path to your certificates folder and the host of the *Stratio GenAI Developer Proxy*.
```

Comment:
We have to check the env generated to see if all the variables are actually needed, and I think it would be nice to explain why the variable is needed (i. e., which component of GenAI core expects it to be defined)